src/styles/blocks/cart.css

```diff
@@ -0,0 +1,104 @@
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-main-color);
+}
+
+.cart {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--background-color);
+  border: solid #f3f3f3 1px;
+  border-radius: 5px;
+}
+
+.cart__image {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.cart__title {
+  margin-bottom: 4px;
+}
+
+.code {
+  line-height: 14px;
+  color: var(--text-second-color);
+}
+
+.cart__code {
+  align-self: flex-start;
+  margin-bottom: 16px;
+}
+
+.reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.cart__reviews {
+  margin-bottom: 24px;
+}
+
+.price {
+  font-style: 12px;
+  line-height: 18px;
+}
+
+.cart__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.price_name {
+  color: var(--text-second-color);
+}
+
+.price_value {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.cart__button {
+  display: flex;
+  justify-content: center;
+
+  font-weight: 700;
+  line-height: 16px;
+  font-size: 14px;
+  color: var(--background-color);
+
+  background-color: var(--button-color);
+  transition: background-color 0.3s ease;
+  border: solid transparent 1px;
+  border-radius: 5px;
+}
+
+.cart__link {
+  display: block;
+  padding: 11px;
+
+  color: inherit;
+  text-transform: uppercase;
+  text-decoration: none;
+  transition: color 0.3s;
+}
+
+.cart__button:hover {
+  background-color: var(--background-color);
+  border: solid var(--button-color) 1px;
```

Comment:
![image](https://user-images.githubusercontent.com/104728827/185438111-370626ea-9d98-4baf-bad4-fdd10c3ac8c2.png)
you already have a transparent border, on hover u just change color